collector/textfile.go

```diff
@@ -213,14 +212,15 @@ fileLoop:
 			error = 1.0
 			continue
 		}
+
+		foundUnsupportedTimestamps := checkUnsupportedTimestamps(parsedFamilies, path)
```

Comment:
Long variable name for a single use right below :)

I'd recommend just inlining this:

`if checkUnsupportedTimestamps(...) {`

Maybe also rename the function to `hasTimestamps()`, makes it clearer what the function actually does?.

Reply:
@juliusv ok thx . ( i am actually a fun of explicit names, i know that is not the golang credo but i tend to prefer file for `f` , in longterm i appreciate this then shorter name (:smile: )

I will inline and change the function name.

If you have any proposal for the variable feel free :+1: :white_flower: 
`UnsuppTimestamps`? or even more shorter?  :smile: tia

Reply:
Since the variable will no longer exist, it doesn't need a name anymore :) Otherwise I would have named it `hasTimestamps` or something (like the function, since it just returns a boolean value, this is the clearest).

Reply:
Ah yes i forgot the inline true :)